woocommerce/woocommerce-android
# [REST API] Show JP banner and simplified JP dialog for REST API logins

Closes: #8595


### Description


For REST API logins, this PR makes the Jetpack bottom banner on My Store screen shows up. When opened, the screen shows the benefits of installing Jetpack (unlocking various features in the app), and adds a button to allow Jetpack install thru a browser.

This is meant to inform users further that some features require Jetpack, and gives them the option to do so. In the future iteration, the installation process will be native instead of thru browser, so this is a temporary solution.


### Testing instructions


1. Prepare a .org test site that has no Jetpack plugin installed,
2. Start app, enter the test site's address, then credentials,
3. After being logged in, make sure the Jetpack banner is shown at the bottom of My Store screen.
4. Tap the banner, make sure the Jetpack benefits dialog is shown with two buttons on it ("Open wp-admin" and "Not now")

Test the "Open wp-admin" button
1. Tap the button, ensure a Chrome Custom Tab opens with a Jetpack setup screen loaded,
2. (optional) follow through the setup process until it Jetpack is installed and connected, the browser will then return to the app and it loads the WPCom login screen.

Test the "Not now" button
1. Tap the button, ensure that the dialog is closed.

### Images/gif


| Banner is shown | Dialog | Chrome Custom Tab |
|-|-|-|
| ![jp-banner](https://user-images.githubusercontent.com/266376/227120934-b3fa1cc9-3db3-4869-be99-0997b081f157.png) | ![jp-dialog](https://user-images.githubusercontent.com/266376/227120931-1c2a6b16-1f69-4550-a8e7-ec067f69ddfc.png) | ![jp-cct-install](https://user-images.githubusercontent.com/266376/227120915-ec88a6b0-43cb-4c07-9719-6ee2d461b9e5.png) |
